package.json

```diff
@@ -98,7 +98,9 @@
   },
   "jest": {
     "testPathIgnorePatterns": [
-      "<rootDir>/frontend/"
+      "<rootDir>/frontend/",
```

Comment:
Ah these should be on a new section `coveragePathIgnorePatterns`, otherwise they'll still be included.
https://jestjs.io/docs/configuration#coveragepathignorepatterns-arraystring

Reply:
Ah, thanks for pointing that out.